dshr/dshr_mod.F90

```diff
@@ -224,7 +224,6 @@ subroutine dshr_mesh_init(gcomp, nullstr, logunit, compname, model_nxg, model_ny
 
     ! local variables
     type(ESMF_VM)                  :: vm
-    type(ESMF_Calendar)            :: esmf_calendar           ! esmf calendar
```

Comment:
Nag complains about variable with same name as module, this isn't used anywhere.